assets/style/works/works.scss

```
@@ -5,3 +5,333 @@
+          width: 70%;
+        }
+        @media (max-width: $phone-l) {
+          display: none;
+          width: 100%;
+        }
+      }
+    }
+    .main__wrapper-common-why-text {
+      margin-top: 55px;
+      width: 50%;
+      @media (max-width: $laptop-l) {
+        width: 50%;
+      }
+      @media (max-width: $phone-l) {
+        margin-top: 10px;
+      }
+      .common-text-heading {
+        margin-bottom: 44px;
+      }
+      .main__wrapper-common-best-wrap {
+        display: flex;
+        flex-wrap: wrap;
+        .main__wrapper-common-best-item {
+          display: flex;
+          justify-content: start;
+          align-items: start;
+          margin-bottom: 50px;
+          .main__wrapper-common-best-item-picture {
+            margin-right: 15px;
+          }
+        }
+      }
+    }
+  }
+  .main__container-additional {
+    max-width: 1920px;
+    width: 100%;
+    margin: 0 auto;
+    text-align: center;
+    background-image: url(../../images/works/blue-crystal-background.png);
+    min-height: 905px;
+    @media (max-width: $pc-m) {
+      margin-bottom: 40px;
+    }
+    .main__link {
+      background-color: $btn-link-dark-color;
+      padding: 16px 54px 19px 55px;
+      text-decoration: none;
+      font-family: $font;
+      font-weight: 700;
+      font-size: 18px;
+      line-height: 21px;
+      color: $text-light-color;
+    }
+    .common-text-heading {
+      padding-top: 110px;
+      margin-bottom: 39px;
+      @media (max-width: $phone-l) {
+        padding-top: 5px;
+      }
+    }
+    .main__wrapper-projecting {
+      display: flex;
+      justify-content: space-between;
+      margin-bottom: 56px;
+      @media (max-width: $pc-m) {
+        flex-wrap: wrap;
+        justify-content: space-around;
+      }
+      @media (max-width: $phone-l) {
+        margin-bottom: 20px;
+      }
+      .main__wrapper-projecting-item {
+        @media (max-width: $phone-l) {
+          width: 300px;
+        }
+        .main__wrapper-projecting-item-img {
+          @media (max-width: $phone-l) {
+            width: 300px;
+          }
+        }
+      }
+      .main__wrapper-projecting-item-text {
+        background-color: white;
+        max-width: 508px;
+        min-height: 224px;
+        display: flex;
+        flex-direction: column;
+        justify-content: start;
+        align-items: start;
+        padding-left: 32px;
+        .main__wrapper-projecting-item-date {
+          background-color: #f1f4f8;
+          font-family: $font;
+          font-weight: 400;
+          font-size: 12px;
+          line-height: 26px;
+          color: $text-blue-color;
+          margin: 20px 0 14px 0;
+        }
+        .main__wrapper-projecting-item-heading {
+          max-width: 350px;
+          min-height: 66px;
+          font-family: $font;
+          font-weight: 700;
+          font-size: 24px;
+          line-height: 26px;
+          color: $text-black-color;
+          text-align: start;
```

Лучше использовать flex-start, если мы работает с flex
```suggestion
          text-align: flex-start;
```